package.json

```diff
@@ -47,6 +47,7 @@
     "@tippyjs/react": "^4.2.5",
     "@types/react-table": "^7.0.18",
     "classnames": "^2.2.6",
+    "react-string-replace": "^0.4.4",
```

Comment:
We try to avoid external dependencies unless they are:
1. reliable, stable, and popular
2. necessary

react-string-replace is pre-1.0 ("unstable") and has less than 400 stars (not very "popular"). More importantly, it requires lodash, which is a huge dependency.

Reply:
That's totally fine, I did not see the transitive lodash dependency and was mostly porting this implementation from another app that uses React table.
I will draft the PR until I remove the dependency.